### PR TITLE
lazySnapshots

### PR DESCRIPTION
**Pull request by pavel-krivanek**
create snapshots of only dirty MC pacakges for faster commits

### CODE REVIEW

**Comment by estebanlm** [comment]
this was not enough to fix the "added package" problem, isn't?

File: Iceberg-UI.package/IceRepositoriesBrowser.class/instance/addPackageToRepository..st [hunk 1]
```diff
@@ -9,4 +9,6 @@ addPackageToRepository: repo
 	options := (MCWorkingCopy allManagers difference: repo workingCopies).
 
 	(self class chooseFrom: options displaying: descriptionBlock) 
-		ifNotNil: [ :workingCopy | repo addPackageNamed: workingCopy packageName ]
\ No newline at end of file
+		ifNotNil: [ :workingCopy | 
+			workingCopy modified: true.
```
Comment (estebanlm) [inline Iceberg-UI.package/IceRepositoriesBrowser.class/instance/addPackageToRepository..st]: I would prefer this logic extracted to a method other than start putting this inside the block.

**Comment by pavel-krivanek** [comment]
no, but because the commit issue of a new package  is common with the mainstream version, it should be fixed separately.

**Comment by estebanlm** [comment]
anyway, according to Nico this was not enough either because this is used to also compare any snapshot with current sources... @npasserini ?

**Comment by pavel-krivanek** [comment]
what about to provide a new UI command, something like Synchronize dirty packages

**Comment by estebanlm** [comment]
this needs to be reimplemented :P

**Pull request closed without merging**